core/src/test/java/tech/tablesaw/io/csv/UnicodeBomHandlingTest.java

```diff
@@ -22,6 +23,7 @@
   }
 
   @Test
+  @Disabled
```

Comment:
Maybe can we change it to read a File instead so that we at least ensure that case is working?

Reply:
Maybe I'm wrong but I really think that the problem is not the test.
It's actually `tablesaw` code that fails to manage BOM in non Unicode environments.

Because if there isn't any management code in `tablesaw` code itself the management that does `univocity`, to my understanding,  only works if we call the parser using a File.
But actual code never uses that method and always preprocess the File itself and passes a reader.

To correct that I think that it will be better to create an issue and manage it in another PR because it will require some refactors on file processing and cache system used.

If my diagnostic is correct I think that disabling the test is better because it lies.

Reply:
Ah, ok. I forgot we didn't call the univocity file method even when the input is a file. That makes sense then